profile-widget/qmlprofile.cpp

```diff
@@ -113,6 +127,34 @@ void QMLProfile::setDevicePixelRatio(qreal dpr)
 	}
 }
 
+qreal QMLProfile::xOffset() const
```

Comment:
Do we really need the getter function? Couldn't we simply declare the PROPERTY as MEMBER?

I'm not a fan of getter functions that do nothing.

Reply:
I thought that if I do `MEMBER` then I can have neither setter nor getter. And since the setter does something other than update the value, this seemed like the way to go. And yes, we need to be able to read this value from QML, so we can't make it write-only :)

Reply:
We do have a few cases of `MEMBER` + `WRITE`:
```
$ git grep MEMBER | grep WRITE
mobile-widgets/qmlmanager.h:    Q_PROPERTY(bool locationServiceEnabled MEMBER m_locationServiceEnabled WRITE setLocationServiceEnabled NOTIFY locationServiceEnabledChanged)
mobile-widgets/qmlmanager.h:    Q_PROPERTY(bool locationServiceAvailable MEMBER m_locationServiceAvailable WRITE setLocationServiceAvailable NOTIFY locationServiceAvailableChanged)
mobile-widgets/qmlmanager.h:    Q_PROPERTY(bool loadFromCloud MEMBER m_loadFromCloud WRITE setLoadFromCloud NOTIFY loadFromCloudChanged)
mobile-widgets/qmlmanager.h:    Q_PROPERTY(QString startPageText MEMBER m_startPageText WRITE setStartPageText NOTIFY startPageTextChanged)
mobile-widgets/qmlmanager.h:    Q_PROPERTY(bool verboseEnabled MEMBER m_verboseEnabled WRITE setVerboseEnabled NOTIFY verboseEnabledChanged)
mobile-widgets/qmlmanager.h:    Q_PROPERTY(QString notificationText MEMBER m_notificationText WRITE setNotificationText NOTIFY notificationTextChanged)
mobile-widgets/qmlmanager.h:    Q_PROPERTY(QString progressMessage MEMBER m_progressMessage WRITE setProgressMessage NOTIFY progressMessageChanged)
mobile-widgets/qmlmanager.h:    Q_PROPERTY(bool btEnabled MEMBER m_btEnabled WRITE setBtEnabled NOTIFY btEnabledChanged)
mobile-widgets/qmlmanager.h:    Q_PROPERTY(bool showNonDiveComputers MEMBER m_showNonDiveComputers WRITE setShowNonDiveComputers NOTIFY showNonDiveComputersChanged)
mobile-widgets/qmlmanager.h:    Q_PROPERTY(qPrefCloudStorage::cloud_status oldStatus MEMBER m_oldStatus WRITE setOldStatus NOTIFY oldStatusChanged)
mobile-widgets/themeinterface.h:        Q_PROPERTY(QString currentTheme MEMBER m_currentTheme WRITE set_currentTheme NOTIFY currentThemeChanged)
profile-widget/qmlprofile.h:    Q_PROPERTY(int diveId MEMBER m_diveId WRITE setDiveId)
```

Reply:
Cool beans. I'll happily try that and simplify the code then. Update coming in a few minutes, I want to get betas for iOS and Android out with the new profile... :-)